lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,7 +7,7 @@ class Json < Base
 
         def serializable_hash(options = nil)
           options ||= {}
-          { root => Attributes.new(serializer).serializable_hash(options) }
+          { root => Attributes.new(serializer, instance_options).serializable_hash(options) }
```

Comment:
Do we have documented the difference between the two kinds of options and what is available? I know this is kinda beating a dead horse, but it seems to be more and more relevant